addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -346,6 +346,9 @@ var SnippetEditor = Widget.extend({
 
         // Show/hide overlay in preview mode or not
         this.$el.toggleClass('oe_active', show);
+        const $optionWrap = this.$el.find('.o_overlay_options_wrap');
+        const isAtPageTop = show && $optionWrap.length && this.$target.get(0).getBoundingClientRect().top + window.scrollY < 20;
```

Comment:
window.scrollY is always 0 since it is never the window which scrolls... something is wrong here

Reply:
Indeed, this part is useless